pennlabs/platform
# Add user profile pictures

To test S3 functionality:
* Move the AWS settings from the production settings to base (or set to prod mode)
* Set S3 credentials as environment variables or in `~/.aws/credentials`
* Files should show up in the `penn.platform` bucket in `/images`
